privval/ipc_test.go

```diff
@@ -0,0 +1,134 @@
+package privval
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	cmn "github.com/tendermint/tendermint/libs/common"
+	"github.com/tendermint/tendermint/libs/log"
+	"github.com/tendermint/tendermint/types"
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
```

Comment:
Imports should go in the following order seperated by a blank line: `stdlib, external packages, internal packages`.